libctru/include/3ds/errf.h

```diff
@@ -15,15 +15,15 @@ typedef enum {
 	ERRF_ERRTYPE_EXCEPTION    = 3, ///< For exceptions, or more specifically 'crashes'. union data should be exception_data.
 	ERRF_ERRTYPE_FAILURE      = 4, ///< For general failure. Shows a message. union data should have a string set in failure_mesg
 	ERRF_ERRTYPE_LOGGED       = 5, ///< Outputs logs to NAND in some cases.
-} ERRF_ErrType;
+} PACKED ERRF_ErrType;
 
 /// Types of 'Exceptions' thrown for ERRF_ERRTYPE_EXCEPTION
 typedef enum {
 	ERRF_EXCEPTION_PREFETCH_ABORT = 0, ///< Prefetch Abort
 	ERRF_EXCEPTION_DATA_ABORT     = 1, ///< Data abort
 	ERRF_EXCEPTION_UNDEFINED      = 2, ///< Undefined instruction
 	ERRF_EXCEPTION_VFP            = 3, ///< VFP (floating point) exception.
-} ERRF_ExceptionType;
+} PACKED ERRF_ExceptionType;
```

Comment:
This PACKED thing isn't a bugfix, it's telling the compiler to use the minimum size to contain all the enum values. This is default on arm-none-eabi anyway so I'm not entirely sure what we gain from this,

This should be it's own commit detailing what it does and why.